.github/workflows/actions.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will build a golang project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-go
+
+name: Go
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: '1.22'
```

Comment:
We usually declare a matrix of Go versions to run against, so we can run against: current major release, previous major release, and upcoming major release (to catch any problems early). See https://github.com/luno/luno-go/blob/main/.github/workflows/test.yml for a reference 👍

Reply:
Will update thanks